stylesheets/_mixin.focus.scss

```diff
@@ -53,8 +53,11 @@
 
 @mixin pulsar-range-focused {
     border: 0;
-    box-shadow: none;
-    outline: none;
+    border-radius: 0;
+    box-shadow: inset 0 0 0 2px color(black);
+    outline: 3px solid color(border, focus);
+    outline-offset: 0;
+    padding: 0 5px;
```

Comment:
Does this cause a jump on focus?

Reply:
Not a jump as such but the range width does shrink a little on focus but with the focus styling it's barely perceivable. Without the padding the outline almost overlaid the range track.

Reply:
![range focus](https://user-images.githubusercontent.com/756393/77307945-19441880-6cf2-11ea-8b31-dfc208b0928b.gif)

Reply:
Ah that's fine then, I was worried about a vertical jump 👍